packages/app/screens/onboarding/OnboardingStack.js

```diff
@@ -0,0 +1,197 @@
+  const [birthday, setBirthday] = useState('');
+  const [interests, setInterests] = useState('');
+  const [error1, setError1 ] = useState(false);
+  const [error2, setError2 ] = useState(false);
+  const [error3, setError3 ] = useState(false);
+  const [error4, setError4 ] = useState(false);
+
+  const { onboardingCompleted } = useUserContext();
+
+  const handleSubmit = async () => {
+    await AsyncStorage.setItem('isFirstTime', JSON.stringify(false));
+    onboardingCompleted();
+
+    if (!school.trim())
+    {
+      setError1(true);
+    }
+    if (!birthday.trim())
+    {
+      setError2(true);
+    }
+    if (!interests.trim())
+    {
+      setError3(true);
+    }
+    if (!school.trim() || !birthday.trim() || !interests.trim())
+    {
+      setError4(true);
+    }
+    else
+    {
+      navigation.navigate('TabNavigation');
+    }
+  };
+
+  const handleSkip = async () => {
+    navigation.navigate('TabNavigation');
+  };
+
+   return (
+    <View>
+      <ScrollView 
+      pagingEnabled={true} 
+      horizontal={true}
+      showsHorizontalScrollIndicator={false}>
+        <View style={styles.slide}>
+          <Text>Where do you go to school?</Text>
+          <TextInput
+          style={styles.input}
+          placeholder= "Your School"
+          onChangeText={setSchool}
+          value={school}
+          />
+          {error2 ? (
+            <Text style={styles.error}>*Your School is Required</Text>
+            ) : null
+          }
+        </View>
+
+        <View style={styles.slide}>
+          <Text>When's your birthday?</Text>
+          <TextInput
+          style={styles.input}
+          placeholder= "MM/DD/YYYY"
+          onChangeText={setBirthday}
+          value={birthday}
+          />
+          {error2 ? (
+            <Text style={styles.error}>*Your Birthday is Required</Text>
+            ) : null
+          }
+        </View>
+          
+        <View style={styles.slide}>
+          <Text>List some of your interest:</Text>
+          <TextInput
+          style={styles.input}
+          placeholder= "Your Interests"
+          onChangeText={setInterests}
+          value={interests}
+          />
+          {error3 ? (
+            <Text style={styles.error}>*Your Interests are Required</Text>
+            ) : null
+          }
+          <TouchableOpacity style={styles.button} onPress={handleSubmit}>
+          <Text style={styles.buttonText}>Submit</Text>
+          </TouchableOpacity>
+          {error4 ? (
+            <Text style={styles.error}>*Missing Required Responses</Text>
+            ) : null
+          }
+        </View>
+      </ScrollView>
+      <View style={styles.indicatorContainer}>
+        <TouchableOpacity style={styles.skipBtn} onPress={handleSkip}>
+          <Text>SKIP</Text>
+        </TouchableOpacity>
+        <View style={styles.indicator}></View>
+        <View style={styles.indicator}></View>
+        <View style={styles.indicator}></View>
```

Comment:
The page indicators on the bottom should show you what page you're on (maybe with a change of color or size).